tests/test_json_to_parquet.py

```diff
@@ -340,7 +340,7 @@ def glue_test_data(glue_flat_table_location, glue_nested_table_location,
 @pytest.fixture(scope="class")
 def glue_crawler_role(namespace):
     iam_client = boto3.client("iam")
-    role_name=f"{namespace}-pytest-crawler-role"
+    role_name=f"{namespace}-pytest-crawler-role"[:60]
```

Comment:
I think the broader issue is that AWS has these character limits on some of its resource names. Given that we are already using the branch name as a way of defining a unique namespace which we prepend to our resource names and that these character limits could pop up on arbitrary resource types (and we don't want to keep track of that or need to adapt for that in our tests), is there a way we can reduce the length of our branch names while still fitting in the information we want in that branch name? What about keeping just the keywords and using some shorthand? Something like `etl-553-multiple-s3-event-configs`?

Reply:
If that solution works for you, can you revert the change on this line, push to a new branch and open a new PR once you've made any other corrections?